README.md

```diff
@@ -64,15 +64,33 @@ Also check out [@ProjectAntrea](https://twitter.com/ProjectAntrea) on Twitter!
 
 ## Features
 
-Antrea currently supports the following features:
-* IPv4 overlay network for a Kubernetes cluster. Geneve, VXLAN, GRE, or STT can
-be used as the encapsulation protocol.
-* [Kubernetes Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies)
-implementation.
-* [Octant](https://github.com/vmware-tanzu/octant) UI plugin for monitoring
-Antrea components, which publish runtime information as
-[CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
-* [IPsec encyption](/docs/ipsec-tunnel.md) of GRE tunnel traffic.
+* **Kubernetes-native**: Antrea follows best practices to extend the Kubernetes
+  APIs and provide familiar abstractions to users, while also leveraging
+  Kubernetes libraries in its own implementation.
+* **Powered by Open vSwitch**: Antrea relies on Open vSwitch to implement all
+  networking functions, including Kubernetes Service load-balancing, and to
+  enable hardware offloading in order to support the most demanding workloads.
```

Comment:
I feel hardware offloading is less important to mention here. But oo real harm either.